addons/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/DSCAlarmBindingConstants.java

```diff
@@ -0,0 +1,110 @@
+    public static final String ENVISALINK_BRIDGE = "envisalinkbridge";
+    public static final String IT100_BRIDGE = "it100bridge";
+
+    // List of DSC Alarm device types
+    public static final String PANEL = "panel";
+    public static final String PARTITION = "partition";
+    public static final String ZONE = "zone";
+    public static final String KEYPAD = "keypad";
+    
+    // List of all Bridge Thing Type UIDs
+    public final static ThingTypeUID ENVISALINKBRIDGE_THING_TYPE = new ThingTypeUID(BINDING_ID, ENVISALINK_BRIDGE);
+    public final static ThingTypeUID IT100BRIDGE_THING_TYPE = new ThingTypeUID(BINDING_ID, IT100_BRIDGE);
+
+    // List of all DSC Alarm Thing Type UIDs
+    public final static ThingTypeUID PANEL_THING_TYPE = new ThingTypeUID(BINDING_ID, PANEL);
+    public final static ThingTypeUID PARTITION_THING_TYPE = new ThingTypeUID(BINDING_ID, PARTITION);
+    public final static ThingTypeUID ZONE_THING_TYPE = new ThingTypeUID(BINDING_ID, ZONE);
+    public final static ThingTypeUID KEYPAD_THING_TYPE = new ThingTypeUID(BINDING_ID, KEYPAD);
+
+    // List of all Channel IDs
+    public final static String BRIDGE_CONNECTION = "bridge_connection";
+    
+    public final static String PANEL_CONNECTION = "panel_connection";
+    public final static String PANEL_MESSAGE = "panel_message";
+    public final static String PANEL_COMMAND = "panel_command";
+    public final static String PANEL_SYSTEM_ERROR = "panel_system_error";
+
+    public final static String PANEL_TROUBLE_MESSAGE = "panel_trouble_message";
+    public final static String PANEL_TROUBLE_LED = "panel_trouble_led";
+    public final static String PANEL_SERVICE_REQUIRED = "panel_service_required";
+    public final static String PANEL_AC_TROUBLE = "panel_ac_trouble";
+    public final static String PANEL_TELEPHONE_TROUBLE = "panel_telephone_trouble";
+    public final static String PANEL_FTC_TROUBLE = "panel_ftc_trouble";
+    public final static String PANEL_ZONE_FAULT = "panel_zone_fault";
+    public final static String PANEL_ZONE_TAMPER = "panel_zone_tamper";
+    public final static String PANEL_ZONE_LOW_BATTERY = "panel_zone_low_battery";
+    public final static String PANEL_TIME_LOSS = "panel_time_loss";
+
+    public final static String PANEL_TIME = "panel_time";
+    public final static String PANEL_TIME_STAMP = "panel_time_stamp";
+    public final static String PANEL_TIME_BROADCAST = "panel_time_broadcast";
+    public final static String PANEL_FIRE_KEY_ALARM = "panel_fire_key_alarm";
+    public final static String PANEL_PANIC_KEY_ALARM = "panel_panic_key_alarm";
+    public final static String PANEL_AUX_KEY_ALARM = "panel_aux_key_alarm";
+    public final static String PANEL_AUX_INPUT_ALARM = "panel_aux_input_alarm";
+	
+    public final static String PARTITION_STATUS = "partition_status";
+    public final static String PARTITION_ARM_MODE = "partition_arm_mode";
+	public final static String PARTITION_ARMED = "partition_armed";
```

Comment:
The files seem to have a mix of tabs and spaces - please run the automatic formatter over all code, so that all tabs are replaced.